symbolic-sourcemapcache/src/lookup.rs

```diff
@@ -173,12 +173,17 @@ impl<'data> SourceMapCache<'data> {
     pub fn lookup(&self, sp: SourcePosition) -> Option<SourceLocation> {
         let idx = match self.min_source_positions.binary_search(&sp.into()) {
             Ok(idx) => idx,
-            Err(0) => 0,
+            Err(0) => return None,
```

Comment:
@Swatinem can you think of a reason it would make sense to return `0` here? Surely the `Err(0)` case means our input location is unmapped?